navigation/ca_hector/launch/geotiff_mapper.launch

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+
+<launch>
+
+  <arg name="trajectory_source_frame_name" default="/base_link"/>
+  <arg name="trajectory_update_rate" default="4"/>
+  <arg name="trajectory_publish_rate" default="0.25"/>
+  <arg name="map_file_path" default="$(find hector_geotiff)/maps"/>
+  <arg name="map_file_base_name" default="hector_slam_map"/>
```

Comment:
Shouldn't this have the name of the environment? This `env` variable may help you: https://github.com/RoboticaUtnFrba/create_autonomy/blob/kinetic-devel/navigation/ca_move_base/launch/navigate.launch#L6